src/__mocks__/notificationUpdate.js

```diff
@@ -15,3 +15,14 @@ export default (Object.notification = {
   },
```

Comment:
It could be better if you rename your file to **updateNotification** instead of **notificationUpdate**

Reply:
Done

Reply:
Good, I can see it